### PR TITLE
[Rules] Add Origin Rules dashboard instructions

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-3554.

### CODE REVIEW

**Comment by pedrosousa** [comment]
Preview of the updated section:
https://pedro-pcx-3554-rules-origin-.cloudflare-docs-7ou.pages.dev/rules/origin-rules/

File: content/rules/origin-rules/_index.md [hunk 1]
```diff
@@ -1,40 +1,38 @@
 ---
 pcx_content_type: concept
-title: Origin Rules (beta)
+title: Origin Rules
 weight: 6
 layout: single
+meta:
+  title: Origin Rules (beta)
```
Comment (marciocloudflare) [inline content/rules/origin-rules/_index.md:7]: Should beta still be here?

File: content/rules/origin-rules/create-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+weight: 2
+meta:
+  title: Create an Origin Rule in the dashboard
+---
+
+# Create an Origin Rule in the dashboard
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account and domain.
+2. Go to **Rules** > **Origin Rules**.
+3. Click **Create origin rule**.
```
Comment (marciocloudflare) [inline content/rules/origin-rules/create-dashboard.md]:
```suggestion
3. Select **Create origin rule**.
```

File: content/rules/origin-rules/create-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+4. Enter a descriptive name for the rule in **Rule name**.
+5.  Under **When incoming requests match**, define the [rule expression](/firewall/cf-dashboard/edit-expressions/).
```
Comment (marciocloudflare) [inline content/rules/origin-rules/create-dashboard.md]:
```suggestion
5. Under **When incoming requests match**, define the [rule expression](/firewall/cf-dashboard/edit-expressions/).
```

File: content/rules/origin-rules/create-dashboard.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+6. Under **Set origin parameters**, define the [Origin Rule settings](/rules/origin-rules/features/) you wish to change for requests matching the rule expression.
+7.  To save and deploy your rule, click **Deploy**. If you are not ready to deploy your rule, click **Save as Draft**.
```
Comment (marciocloudflare) [inline content/rules/origin-rules/create-dashboard.md]:
```suggestion
7. To save and deploy your rule, select **Deploy**. If you are not ready to deploy your rule, select **Save as Draft**.
```

File: content/rules/origin-rules/faq.md [hunk 5]
```diff
@@ -10,13 +10,13 @@ meta:
 
 Below you will find answers to the most commonly asked questions regarding Origin Rules.
 
-## What happens if I use both an Origin Rule and a Page Rule to perform a Host header/resolve override?
+## What happens if I use both an Origin Rule and a Page Rule to perform a Host header/DNS record override?
 
 In this situation the Origin Rule parameters will override the [Page Rule](https://support.cloudflare.com/hc/en-us/articles/218411427) parameters. Consider the following example scenarios:
 
-* A Page Rule defines a Host header override, but not an resolve override. An Origin Rule defines an resolve override, but not a Host header override. The resulting request will have the `Host` header defined by the Page Rule and the origin hostname defined by the Origin Rule.
+* A Page Rule defines a Host header override, but not a resolve override (or DNS record override). An Origin Rule defines an DNS record override, but not a Host header override. The resulting request will have the `Host` header defined by the Page Rule and the origin hostname defined by the Origin Rule.
```
Comment (marciocloudflare) [inline content/rules/origin-rules/faq.md]:
```suggestion
* A Page Rule defines a Host header override, but not a resolve override (or DNS record override). An Origin Rule defines a DNS record override, but not a Host header override. The resulting request will have the `Host` header defined by the Page Rule and the origin hostname defined by the Origin Rule.
```

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**